src/CRM/Topology.hs

```diff
@@ -68,3 +69,12 @@ $( singletons
       trivialTopology = Topology []
       |]
  )
+
+-- ** Allow all topology
+
+$( singletons
+    [d|
+      allowAllTopology :: (Bounded a, Enum a) => Topology a
+      allowAllTopology = Topology [(a, [minBound .. maxBound]) | a <- [minBound .. maxBound]]
```

Comment:
I wish there was a way to specify this without these constraints.

I also wonder how this topology performs if given a really large set, e.g. `Int`, or even `Integer`. I can conceive of a state machine that uses an Int for state.

Reply:
I've opened #32 to track this.